src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,56 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: $col-count * $cell-size + $col-count - 1;
+  padding: 10px;
+
+  $week:
+  "mon" 0,
+  "tue" 1,
+  "wed" 2,
+  "thu" 3,
+  "fri" 4,
+  "sat" 5,
+  "sun" 6;
+
+  @each $name, $day in $week {
+    &--start-day--#{$name} :first-child {
+      margin-left: $day * $cell-size + $day;
+    }
+  }
+
+  @for $days from 28 through 31 {
+    &--month-length--#{$days} :nth-child(n + #{$days + 1}) {
+      display: none;
+    }
+  }
+
+  &__cell {
+    border: 1px solid #000;
+    width: $cell-size;
+    height: $cell-size;
+    box-sizing: border-box;
+    background-color: #eee;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    &:hover {
+      cursor: pointer;
+      background-color: pink;
+      transform: translateY(-20px);
```

Comment:
use transition property to make your animation smoother

Reply:
@alexandra-protyanova Hello! All bugs were fixed.